skorch/callbacks/scoring.py

```diff
@@ -34,6 +35,19 @@ def cache_net_infer(net, use_caching, y_preds):
     del net.__dict__['infer']
 
 
+def convert_sklearn_metric_function(scoring):
+    """If ``scoring`` is a sklearn metric function, convert it to a
+    sklearn scorer and return it. Otherwise, return ``scoring`` unchanged."""
+    if callable(scoring):
+        module = getattr(scoring, '__module__', None)
+        if (hasattr(module, 'startswith') and
```

Comment:
Could you please change the indentation of the conditions to increase readability:

```
if (
        cond0 and
        cond1 and ...
):
    return ...
```